src/features/share/asImage/DefaultPreferences.ts

```diff
@@ -0,0 +1,16 @@
+import { ShareAsImagePreferences } from "./ShareAsImagePreferences";
+
+export const defaultPreferences: ShareAsImagePreferences = {
```

Comment:
I'd like to have these in a separate file, as I intend to flip some of these values downstream